src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -29,7 +30,12 @@
                         KBase environment (default: PROD)
   --upload_file_ext UPLOAD_FILE_EXT [UPLOAD_FILE_EXT ...]
                         Upload only files that match given extensions (default: ['genomic.fna.gz'])
-  --workers WORKERS     Number of workers for multiprocessing (default: 5)
+  --batch_size BATCH_SIZE
+                        Number of files to upload per batch (default: 1000)
```

Comment:
Why have a batch size anymore? If the script just submits everything to AssyUtil it should handle things, right?

Reply:
yeah, I actually don't need it. 

If the script catch an exception, I can simply list_objects on workspace and find corresponding UPAs of successful ones and do the linking.

Reply:
ah, but If we get rid of batch_size, how can we get the progress bar and see where we are in uploading?

Reply:
Some options:

1) We don't know
2) We stream the logs, which gives us some idea of what's going on
3) The script starts a thread which does a workspace_info call every X (say 30) seconds and reports the number of objects completed (this only works for an empty workspace, so maybe not feasible)
3.5) As above but uses get_object_info to see which objects are done
4) Update AssemblyUtil to write progress to a file, which the wrapper script reads
5) Leave the batch stuff in, which honestly seems a bit silly just to get progress

Reply:
Updated!

Reply:
So am I correct in thinking we're still just uploading in batches to keep track of how much work has been done?

Reply:
yes, I want to keep this batch_size param.

Reply:
Ok. I'm not a huge fan of this approach personally but it'll do for now